typewise_alert.py

```diff
@@ -1,46 +1,42 @@
 
+CoolingTypeLimit = { 'PASSIVE_COOLING': {'lowerLimit': 0,'upperLimit':35},
+                     'HI_ACTIVE_COOLING': {'lowerLimit': 0,'upperLimit':45},
+                     'MED_ACTIVE_COOLING': {'lowerLimit': 0,'upperLimit':40}
+                   }
+
+EMailInfo = { 'TOO_LOW' : {'Recepient':'a.b@c.com','Message':'Hi, the temperature is too low'},
```

Comment:
Facility to have separate email recepients for TOO_LOW and TOO_HIGH: is this intentional?

Reply:
Yes in case we need to send it to different recipient. In this case we are sending it to the same recipient.

Reply:
i have noticed that in my previous code if the user passes the cooling type which are not defined in the "coolingTypeLimit" dictionary then it will throw an error. So to avoid this I have added an additional if statement which returns the alert type only for the cooling type which are available in "coolingTypeLimit" dictionary.
![image](https://user-images.githubusercontent.com/13776900/114032061-96104900-9899-11eb-90cf-3b3fa7064a53.png)
<img width="424" alt="2021-04-08_18h34_49" src="https://user-images.githubusercontent.com/13776900/114032078-990b3980-9899-11eb-83ec-c66867f80601.png">